### PR TITLE
GMP 6.2.1

### PR DESCRIPTION
**Pull request by stikonas**
Ran `make check` afterwards , all tests pass.

### CODE REVIEW

File: sysa/gmp-6.2.1/gmp-6.2.1.sh [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# SPDX-FileCopyrightText: 2021 Andrius Štikonas <andrius@stikonas.eu>
+#
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+src_prepare() {
+    default_src_prepare
+
+    rm demos/calc/calc.{c,h} demos/calc/calclex.c
```
Comment (fosslinux) [inline sysa/gmp-6.2.1/gmp-6.2.1.sh:8]: Why (I assume autogened?) Comment please.

File: sysa/gmp-6.2.1/gmp-6.2.1.sh [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+    libtoolize -f
+    aclocal
+    autoconf
+    automake --add-missing
```
Comment (fosslinux) [inline sysa/gmp-6.2.1/gmp-6.2.1.sh:13]: `autoreconf -f`?

**Review by fosslinux — CHANGES_REQUESTED** [review]

File: sysa/gmp-6.2.1/gmp-6.2.1.sh (hunk 1 above)
Comment (stikonas) [reply sysa/gmp-6.2.1/gmp-6.2.1.sh:8]: yeah, flex and bison.

File: sysa/gmp-6.2.1/gmp-6.2.1.sh (hunk 2 above)
Comment (stikonas) [reply sysa/gmp-6.2.1/gmp-6.2.1.sh:13]: Did not work as it did not rebuild libtools, so we then hit libtool/aclocal incompatibility.

**Review by fosslinux — APPROVED** [review]

File: sysa/gmp-6.2.1/gmp-6.2.1.sh (hunk 2 above)
Comment (stikonas) [reply sysa/gmp-6.2.1/gmp-6.2.1.sh]: autoreconf -fi works

**Pull request merged**